releases/release-notes.md

```diff
@@ -7,189 +7,192 @@ aliases: ['/docs/dev/releases/release-notes/','/docs/dev/releases/rn/']
+- [2.1.16](/releases/release-2.1.16.md): 2019-08-15
+- [2.1.15](/releases/release-2.1.15.md): 2019-07-18
+- [2.1.14](/releases/release-2.1.14.md): 2019-07-04
+- [2.1.13](/releases/release-2.1.13.md): 2019-06-21
+- [2.1.12](/releases/release-2.1.12.md): 2019-06-13
+- [2.1.11](/releases/release-2.1.11.md): 2019-06-03
+- [2.1.10](/releases/release-2.1.10.md): 2019-05-22
+- [2.1.9](/releases/release-2.1.9.md): 2019-05-06
+- [2.1.8](/releases/release-2.1.8.md): 2019-04-12
+- [2.1.7](/releases/release-2.1.7.md): 2019-03-28
+- [2.1.6](/releases/release-2.1.6.md): 2019-03-15
+- [2.1.5](/releases/release-2.1.5.md): 2019-02-28
+- [2.1.4](/releases/release-2.1.4.md): 2019-02-15
+- [2.1.3](/releases/release-2.1.3.md): 2019-01-28
+- [2.1.2](/releases/release-2.1.2.md): 2018-12-22
+- [2.1.1](/releases/release-2.1.1.md): 2018-12-12
+- [2.1.0](/releases/release-2.1-ga.md): 2018-11-30
+- [2.1.0-rc.5](/releases/release-2.1-rc.5.md): 2018-11-12
+- [2.1.0-rc.4](/releases/release-2.1-rc.4.md): 2018-10-23
+- [2.1.0-rc.3](/releases/release-2.1-rc.3.md): 2018-09-29
+- [2.1.0-rc.2](/releases/release-2.1-rc.2.md): 2018-09-14
+- [2.1.0-rc.1](/releases/release-2.1-rc.1.md): 2018-08-24
+- [2.1.0-beta](/releases/release-2.1-beta.md): 2018-06-29
 
 ## 2.0
 
-- [2.0.11](/releases/release-2.0.11.md)
-- [2.0.10](/releases/release-2.0.10.md)
-- [2.0.9](/releases/release-2.0.9.md)
-- [2.0.8](/releases/release-2.0.8.md)
-- [2.0.7](/releases/release-2.0.7.md)
-- [2.0.6](/releases/release-2.0.6.md)
-- [2.0.5](/releases/release-2.0.5.md)
-- [2.0.4](/releases/release-2.0.4.md)
-- [2.0.3](/releases/release-2.0.3.md)
-- [2.0.2](/releases/release-2.0.2.md)
-- [2.0.1](/releases/release-2.0.1.md)
-- [2.0 GA](/releases/release-2.0-ga.md)
-- [2.0 RC5](/releases/release-2.0-rc.5.md)
-- [2.0 RC4](/releases/release-2.0-rc.4.md)
-- [2.0 RC3](/releases/release-2.0-rc.3.md)
-- [2.0 RC1](/releases/release-2.0-rc.1.md)
-- [1.1 Beta](/releases/release-1.1-beta.md)
-- [1.1 Alpha](/releases/release-1.1-alpha.md)
+- [2.0.11](/releases/release-2.0.11.md): 2019-01-03
+- [2.0.10](/releases/release-2.0.10.md): 2018-12-18
+- [2.0.9](/releases/release-2.0.9.md): 2018-11-19
+- [2.0.8](/releases/release-2.0.8.md): 2018-10-16
+- [2.0.7](/releases/release-2.0.7.md): 2018-09-07
+- [2.0.6](/releases/release-2.0.6.md): 2018-08-06
+- [2.0.5](/releases/release-2.0.5.md): 2018-07-06
+- [2.0.4](/releases/release-2.0.4.md): 2018-06-15
+- [2.0.3](/releases/release-2.0.3.md): 2018-06-01
+- [2.0.2](/releases/release-2.0.2.md): 2018-05-21
+- [2.0.1](/releases/release-2.0.1.md): 2018-05-16
+- [2.0.0](/releases/release-2.0-ga.md): 2018-04-27
+- [2.0.0-rc.5](/releases/release-2.0-rc.5.md): 2018-04-17
+- [2.0.0-rc.4](/releases/release-2.0-rc.4.md): 2018-03-30
+- [2.0.0-rc.3](/releases/release-2.0-rc.3.md): 2018-03-23
+- [2.0.0-rc.1](/releases/release-2.0-rc.1.md): 2018-03-09
+
+## 1.1
+
+- [1.1.0-beta](/releases/release-1.1-beta.md): 2018-02-24
+- [1.1.0-alpha](/releases/release-1.1-alpha.md): 2018-01-19
```

Comment:
Let's keep the original place for 1.1 because 1.1 has no GA version.
```suggestion
- [1.1.0-beta](/releases/release-1.1-beta.md): 2018-02-24
- [1.1.0-alpha](/releases/release-1.1-alpha.md): 2018-01-19
```